FeedStoreChallenge/CoreDataFeedStore+FeedStore.swift

```diff
@@ -0,0 +1,54 @@
+//
+//  CoreDataFeedStore+FeedStore.swift
+//  FeedStoreChallenge
+//
+//  Created by Alex Thurston on 5/10/21.
+//  Copyright © 2021 Essential Developer. All rights reserved.
+//
+
+import Foundation
+import CoreData
+
+extension CoreDataFeedStore: FeedStore {
+	public func retrieve(completion: @escaping RetrievalCompletion) {
+		perform { context in
+			do {
+				guard let feed = try CoreDataFeed.fetch(context: context) else {
+					completion(.empty)
+					return
+				}
+				completion(.found(feed: feed.localFeedImages, timestamp: feed.timestamp))
+			} catch {
+				completion(.failure(error))
+			}
+		}
+	}
+
+	public func insert(_ feed: [LocalFeedImage], timestamp: Date, completion: @escaping InsertionCompletion) {
+		perform { context in
+			do {
+				let coreDataFeed = try CoreDataFeed.createUniqueFeed(context: context)
+				let images = CoreDataFeedImage.images(feed: feed, context: context)
+				coreDataFeed.feedImages = images
```

Comment:
Following the comment on `CoreDataFeed` please rename `feedImages` to `images`.

```suggestion
				coreDataFeed.images = images
```